### PR TITLE
Fixing issue #1 + adding the ability to have a preview of the image o…

### PR DESCRIPTION
**Pull request by clemblanco**
Fixing issue #1 + adding the ability to have a preview of the image once saved and a delete button as well.

### CODE REVIEW

**Comment by ceckoslab** [comment]
Thanks! I will check your PR. I assume that this JS error could appear in other chooser create function, so I will see if it's needed to move your fix in another class.

**Comment by ceckoslab** [comment]
I like the idea in this PR, but I tried it locally and it didn't work at all. I got 404 page when I tried to show the image upload dialog. The reason is the url refers to controller defined in https://github.com/aijko/aijko-widgetimagechooser My extension is standalone and it already solves some problems that https://github.com/aijko/aijko-widgetimagechooser solved by adding new controller. My solution is better, because it solves the problem with 5 lines of JS. 

Basically if I merge your PR then this extension should always work with together with https://github.com/aijko/aijko-widgetimagechooser. I think that I can reuse 95% of your code in combination with my JS solution.

Do you mind if I put some comments in your PR? Or I just can correct what's needed on my own?

Thanks!

**Comment by clemblanco** [comment]
Oops my bad didn't realize I was calling the other extension controller. Of course it shouldn't.
Feel free to modify the PR as you need or I could even do it if you want.

**Comment by clemblanco** [comment]
Yeah you're right this bit `$chooserUrl = $this->getUrl('*/cms_wysiwyg_images_chooser/index', array('target_element_id' => $elementId));` is using the other extension... Hum... Not sure what's best to do here. Your call.

**Comment by ceckoslab** [comment]
@Claymm I am about to close this PR because I did my implementation based on #3 My changes are already in master branch. You can check if this works for you.

Thanks for your idea!

**Comment by ceckoslab** [comment]
Hello @Claymm I am wondering if you had a chance to try my recent changes. It looks more user friendly now :)
